src/main/java/org/hawkular/client/android/activity/LoginActivity.java

```diff
@@ -126,6 +129,48 @@ public void login() {
         }
     }
 
+    @OnClick(R.id.button_QR)
+    public void scan() {
+        IntentIntegrator integrator = new IntentIntegrator(LoginActivity.this);
+        integrator.initiateScan();
+    }
+
+
+    public void onActivityResult(int requestCode, int resultCode, Intent response) {
+        IntentResult scanResult = IntentIntegrator.parseActivityResult(requestCode, resultCode, response);
+        if (scanResult != null && scanResult.getContents() != null) {
+            BackendClient.of(this).configureAuthorization(this.getApplicationContext());
+            String authorization = scanResult.getContents();
+            String[] authArray = authorization.split(",");
+            if (authArray.length >= 2) {
+                try {
+                    if (!isPortAvailable()) {
+                        backendUrl = Urls.getUrl(getHost());
+                    } else {
+                        backendUrl = Urls.getUrl(getHost(), getPortNumber());
+                    }
+                    username = edit_username.getText().toString();
+                    password = edit_password.getText().toString();
+                    BackendClient.of(this).configureAuthorization(getApplicationContext());
+                    BackendClient.of(this).deauthorize();
+                    Intent intent = this.getIntent();
+                    intent.putExtra("key", authArray[0]);
+                    intent.putExtra("secret", authArray[1]);
+                    intent.putExtra("url", backendUrl.toString());
+                    intent.putExtra("contain", "pair");
+                    if(authArray.length==3)
```

Comment:
Here you have the opening brace on the next line which is not consistent 
with other places


Am 22. März 2016 5:49:18 nachm. schrieb Anuj Garg <notifications@github.com>:

>> +                try {
>> +                    if (!isPortAvailable()) {
>> +                        backendUrl = Urls.getUrl(getHost());
>> +                    } else {
>> +                        backendUrl = Urls.getUrl(getHost(), getPortNumber());
>> +                    }
>> +                    username = edit_username.getText().toString();
>> +                    password = edit_password.getText().toString();
>> +                    
>> BackendClient.of(this).configureAuthorization(getApplicationContext());
>> +                    BackendClient.of(this).deauthorize();
>> +                    Intent intent = this.getIntent();
>> +                    intent.putExtra("key", authArray[0]);
>> +                    intent.putExtra("secret", authArray[1]);
>> +                    intent.putExtra("url", backendUrl.toString());
>> +                    intent.putExtra("contain", "pair");
>> +                    if(authArray.length==3)
>
> Sorry I am not getting you
>
> ---
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/hawkular/hawkular-android-client/pull/56/files/d6253e4c642c4af35836f04dd40185c646bc35ab#r57034996